tests/aws/services/sns/test_sns.py

```diff
@@ -4638,7 +4638,10 @@ def test_subscription_tokens_can_retrospect(
             timeout=2,
         )
 
-        wrong_sub_arn = subscription_arn.replace(TEST_AWS_REGION_NAME, "us-west-1")
+        wrong_sub_arn = subscription_arn.replace(
+            TEST_AWS_REGION_NAME,
+            "us-west-1" if TEST_AWS_REGION_NAME != "us-west-1" else "us-east-1",
+        )
```

Comment:
Maybe use an uncommon region instead?

```suggestion
        wrong_sub_arn = subscription_arn.replace(TEST_AWS_REGION_NAME, "me-south-1")
```

Reply:
maybe this `"il-central-1" if TEST_AWS_REGION_NAME != "il-central-1" else "me-south-1"` would be better @viren-nadkarni? The check seems necessary.